### PR TITLE
Add notebook example and setup binder

### LINKED ISSUE #58: Add Jupyter notebook demo

### PR DESCRIPTION
**Pull request by fcollonval**
Fix #58 

- [x] Binder badge needs to be update to point to the master branch in this repo before merging.

### CODE REVIEW

File: examples/demoslave.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        self.positive_pin_i = 0.001
+        self.negative_pin_v = 10.
+        self.negative_pin_i = 0.001
+        self.delta_v = 10.
+        self.i = 0.001
+        self.R = 10000.
+        
+        self.register_variable(Real("R", causality=Fmi2Causality.parameter))
+
+        self.register_variable(Real("positive_pin_v", causality=Fmi2Causality.input, variability=Fmi2Variability.tunable))
```
Comment (markaren) [inline examples/demoslave.py]: Remove variability from inputs. While I do not think the standard prohibits it, I think it only makes noise to add it to an input.

File: examples/demoslave.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        self.positive_pin_v = 20.
+        self.positive_pin_i = 0.001
+        self.negative_pin_v = 10.
+        self.negative_pin_i = 0.001
+        self.delta_v = 10.
+        self.i = 0.001
+        self.R = 10000.
+        
+        self.register_variable(Real("R", causality=Fmi2Causality.parameter))
```
Comment (markaren) [inline examples/demoslave.py]: This parameter on the other hand should be marked `tunable` as you state users may change it.

**Comment by fcollonval** [comment]
Thanks for the review @markaren 

Is it good to go for you?

**Comment by markaren** [comment]
> In particular I don't know if the position of the binder badge in the README is appropriate.

No, It looks really ugly :P

How about:

....

###### Create the FMU
...

###### Test it online
<badge goes here>

**Comment by markaren** [comment]
Also the notebook should show the step installing pythonfmu though pip

**Comment by markaren** [comment]
I still think the `Test it on` text renders ugly. Enough to just have the badge there?

**Comment by markaren** [comment]
I'm gonna push some minor changes in a few.

File: examples/demo.ipynb [hunk 3]
```diff
@@ -253,7 +253,7 @@
     "p.plot(res['time'], res['i'], res2['time'], res2['i'])\n",
     "p.xlabel('time')\n",
     "p.ylabel('Voltage [V]')\n",
-    "p.show();"
+    "p.show()"
```
Comment (markaren) [inline examples/demo.ipynb]: Was it ok to remove this here ? @fcollonval

Comment (markaren) [reply examples/demo.ipynb]: Reverted just to be safe. merging this

**Pull request merged**